semaphore.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```

Comment:
Can you add a comment to explain similar to 
https://github.com/weaveworks/tcptracer-bpf/blob/master/semaphore.sh